Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'manageiq-password',   '~> 0.2', ">= 0.2.1"
 gem 'mimemagic',           '~> 0.3.3'
 gem 'more_core_extensions'
 gem 'pg',                  '~> 1.0', :require => false
-gem 'puma',                '~> 3.0'
```

Comment:
Do we need to lock down to 3.12?  Is `~>3.0, >=3.12.2` ok?

Reply:
I think that would work too.  was trying to be consistent with sources.

Reply:
Somehow Hakiri did not like ~>3.0, >=3.12.2,  moving to just >= 3.12.2 worked.

Reply:
If I remember right hakiri wanted `>= 3.12.2` first, then `~> 3.0`

Reply:
Ahh, lemme try that.

Reply:
Wow really can't make hakiri happy...

Reply:
holy cow, REALLY picky, tried '>= 3.12.2' with '~> 3.0', then '~> 3.12', and finally with '~> 3.12.2', still failed.  looks like the only one that works is the original one like sources, i.e. just '~> 3.12.2'